ui/i18n/messages.json

```
@@ -81,6 +81,16 @@
   "documentAclTable.until": "Until",
   "documentAclTable.untilMiddle": "until",
   "documentAclTable.userGroup": "User / Group",
+  "documentContent.moveToColdStorage.ask.confirmation":"Send to cold storage?",
+  "documentContent.moveToColdStorage.cancel":"Cancel",
+  "documentContent.moveToColdStorage.confirm":"Confirm",
+  "documentContent.moveToColdStorage.error":"Error moving the document content to cold storage.",
+  "documentContent.moveToColdStorage.error.cannotPerformMove":"You cannot move the document content to cold storage.",
+  "documentContent.moveToColdStorage.label":"Send to cold storage",
+  "documentContent.moveToColdStorage.ok":"OK",
+  "documentContent.moveToColdStorage.description":"This will result in increased file retrieval times and a low resolution preview.",
+  "documentContent.moveToColdStorage.success":"Document content moved to cold storage successfully.",
+  "documentContent.moveToColdStorage.tooltip":"Send to cold storage",
```

All these labels are missing a space after the ":"
```suggestion
  "documentContent.moveToColdStorage.tooltip": "Send to cold storage",
```